app/controllers/works_controller.rb

```diff
@@ -39,6 +42,10 @@ def edit
 
     @form = WorkForm.new(work)
     @form.prepopulate!
+
+    add_breadcrumb title: work.collection.name, link: collection_path(work.collection), confirm: true
+    add_breadcrumb title: work.title, link: work_path(work), confirm: true
+    add_breadcrumb title: 'Edit'
```

Comment:
This is not in the design: https://projects.invisionapp.com/share/EQXC9CLKCR2#/screens/420954309

Reply:
I got this from the Design Doc: https://docs.google.com/document/d/1fcr2DYo7OrX-qTdeUOTWKSS1rlq_47sehESmbvgUsrk/edit#  But you are right, it is not consistent with the Wireframe.  @astridu ?